modules/carousel/jetpack-carousel.php

```diff
@@ -216,22 +216,22 @@ function enqueue_assets() {
 				'single_image_gallery' => $this->single_image_gallery_enabled,
 				'single_image_gallery_media_file' => $this->single_image_gallery_enabled_media_file,
 				'background_color'     => $this->carousel_background_color_sanitize( Jetpack_Options::get_option_and_ensure_autoload( 'carousel_background_color', '' ) ),
-				'comment'              => __( 'Comment', 'jetpack' ),
-				'post_comment'         => __( 'Post Comment', 'jetpack' ),
-				'write_comment'        => __( 'Write a Comment...', 'jetpack' ),
-				'loading_comments'     => __( 'Loading Comments...', 'jetpack' ),
+				'comment'              => esc_html__( 'Comment', 'jetpack' ),
+				'post_comment'         => esc_html__( 'Post Comment', 'jetpack' ),
+				'write_comment'        => esc_html__( 'Write a Comment...', 'jetpack' ),
+				'loading_comments'     => esc_html__( 'Loading Comments...', 'jetpack' ),
```

Comment:
I'm afraid that this is too early for escaping. For security reasons it is better to start with converting client side scripts to use DOM operations instead of concatenating strings. Plus, it's hard to imagine HTML appearing in these translations.